### PR TITLE
#1878 fix husky

### PR DESCRIPTION
**Pull request by porcelain11**
Re-install husky, add prettier:write to pre-commit hook

### CODE REVIEW

**Review by KonstantinEpam23 — APPROVED** [review]

File: packages/ketcher-react/src/script/ui/state/hotkeys.ts [hunk 1]
```diff
@@ -71,7 +72,7 @@ function keyHandle(dispatch, state, hotKeys, event) {
       const newAction = actions[actName].action
       dispatch(onAction(newAction))
       event.preventDefault()
-    } else if (window.clipboardData) {
+    } else if (isIE) {
       // IE support
```
Comment (Nitvex) [inline packages/ketcher-react/src/script/ui/state/hotkeys.ts:76]: Propose to remove this comment

File: package.json [hunk 2]
```diff
@@ -39,5 +39,8 @@
   "resolutions": {
     "react-error-overlay": "6.0.9"
   },
-  "packageManager": "yarn@3.1.0"
+  "packageManager": "yarn@3.1.0",
+  "dependencies": {
```
Comment (Nitvex) [inline package.json:43]: Propose to move this dependency to `ketcher-react`

**Review by Nitvex — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/script/ui/state/hotkeys.ts (hunk 1 above)
Comment (porcelain11) [reply packages/ketcher-react/src/script/ui/state/hotkeys.ts]: Fixed

File: package.json (hunk 2 above)
Comment (porcelain11) [reply package.json:42]: Fixed

**Review by Nitvex — APPROVED** [review]

**Pull request merged**